pkgs/by-name/pa/parabolic/package.nix

```
@@ -1,77 +1,119 @@
-{ lib
-, buildDotnetModule
-, fetchFromGitHub
-, dotnetCorePackages
-, gtk4
-, libadwaita
-, pkg-config
-, wrapGAppsHook4
-, glib
-, shared-mime-info
-, gdk-pixbuf
-, blueprint-compiler
-, python3
-, ffmpeg
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  gettext,
+  itstool,
+  ninja,
+  yelp-tools,
+  pkg-config,
+  libnick,
+  boost,
+  glib,
+  shared-mime-info,
+  gtk4,
+  libadwaita,
+  wrapGAppsHook4,
+  libxmlxx5,
+  blueprint-compiler,
+  qt6,
+  yt-dlp,
+  ffmpeg,
+  aria2,
+  nix-update-script,
+  useQtInterface ? false,
 }:
 
```

```suggestion

assert lib.assertOneOf "uiPlatform" uiPlatform [ "gnome" "qt" ];

```
We can also type check it